tests/libnet/cluster.go

```diff
@@ -44,3 +47,11 @@ func getPodByKubeVirtRole(virtClient kubecli.KubevirtClient, kubevirtPodRole str
 	}
 	return &pods.Items[0], nil
 }
+
+func SkipWhenNotDualStackCluster(virtClient kubecli.KubevirtClient) {
```

Comment:
Could you find another home for the `skip`? I really think the gomega / ginkgo dependencies in this file are weird - this was meant to house extracting info from the cluster.

Reply:
Where? utils.go?:)
Couldn't find better place under our new network folders.

Reply:
I will create skips.go file under libnet.

Reply:
Thanks!